tests/conftest.py

```diff
@@ -472,12 +479,19 @@ def pagerduty_integration(request, pagerduty_service):
     Managed Service.
 
     """
-    if config.ENV_DATA[
-        "platform"
-    ].lower() not in constants.MANAGED_SERVICE_PLATFORMS or config.ENV_DATA.get(
-        "disable_pagerduty"
+    teardown = config.RUN["cli_params"]["teardown"]
+    skip_ocs_deployment = config.ENV_DATA["skip_ocs_deployment"]
+    ceph_cluster_installed = config.RUN.get("cephcluster")
+    if teardown or skip_ocs_deployment or not ceph_cluster_installed:
+        log.info("CephCluster is not available. Skipping PagerDuty integration.")
+        return None
+    if (
+        config.ENV_DATA["platform"].lower() not in constants.MANAGED_SERVICE_PLATFORMS
+        or config.ENV_DATA.get("disable_pagerduty")
+        or (teardown or skip_ocs_deployment or not ceph_cluster_installed)
```

Comment:
Haven't you already taken care of these cases by returning None on line 487?

Reply:
Yes, updated